js/types/objects.js

```diff
@@ -105,7 +114,17 @@ export class MCParticle extends EDMObject {
     const topLines = [];
     topLines.push("ID: " + this.index);
     topLines.push("Gen. stat.: " + this.generatorStatus);
-    topLines.push("Sim. stat.: " + this.simulatorStatus);
+    const simulatorStatus = getSimStatusDisplayValuesFromBit(
+      this.simulatorStatus
+    );
+    const simulatorStatusFirstLetter = simulatorStatus
+      .map((s) => s[0])
+      .join(", ");
```

Comment:
```suggestion
      .join("");
```
As "explained" in the top level comment.